Source/WebCore/dom/ContentVisibilityDocumentState.cpp

```diff
@@ -96,50 +94,82 @@ IntersectionObserver* ContentVisibilityDocumentState::intersectionObserver(Docum
     return m_observer.get();
 }
 
-DidUpdateAnyContentRelevancy ContentVisibilityDocumentState::updateRelevancyOfContentVisibilityElements(const OptionSet<ContentRelevancy>& relevancyToCheck)
+bool ContentVisibilityDocumentState::checkRelevancyOfContentVisibilityElement(Element& target, const OptionSet<ContentRelevancy>& relevancyToCheck) const
```

Comment:
`OptionSet<>` is small enough to be passed by value.

Reply:
Fixed.